### PR TITLE
feat: forward disabled property to underlying button

### PR DESCRIPTION
**Pull request by frabbit**
- forward disabled property to underlying button
- this PR removes `onClickDisabled` which does not work in that case anymore. (so this is breaking).

### CODE REVIEW

File: src/components/Button/Button.js [hunk 1]
```diff
@@ -14,15 +14,14 @@ const Button = ({
...
-  const onClickProp = disabled ? onClickDisabled : onClick;
 
   return (
     <Element
-      onClick={onClickProp}
+      onClick={onClick}
+      {...(Element === 'button' && disabled ? { disabled } : {})}
       {...props}
       className={classnames(
         olt.Button,
```
Comment (drinchev) [inline src/components/Button/Button.js]: Btw I just saw that we don't have `olt.ButtonSelectable` class name in the style guide. Let's remove it.

File: src/components/Button/Button.js [hunk 2]
```diff
@@ -14,15 +14,14 @@ const Button = ({
...
   ...props
 }) => {
   const Element = tag || 'button';
-  const onClickProp = disabled ? onClickDisabled : onClick;
 
   return (
     <Element
-      onClick={onClickProp}
+      onClick={onClick}
+      {...(Element === 'button' && disabled ? { disabled } : {})}
```
Comment (drinchev) [inline src/components/Button/Button.js]: If we do have `disabled` attribute, then we don't need the `olt.isDisabled` class added to the class names.

File: src/components/Button/Button.js (hunk 1 above)
Comment (frabbit) [reply src/components/Button/Button.js]: You're totally right, i have no idea actually. It looks to me like a copy and paste thingy from the past. `selectable` for Buttons is not used in the portal at all.

File: src/components/Button/Button.js (hunk 2 above)
Comment (frabbit) [reply src/components/Button/Button.js]: I removed that class, but we need this merged and release first: https://github.com/lightelligence-io/styles/pull/5. There was a missing selector for the `:disabled` pseudo selector.

**Comment by drinchev** [comment]
@frabbit we need to resolve conflicts here ;)

**Pull request merged**